framework/helpers/BaseInflector.php

```diff
@@ -504,7 +504,13 @@ public static function transliterate($string, $transliterator = null)
                 $transliterator = static::$transliterator;
             }
 
-            return transliterator_transliterate($transliterator, $string);
+            if (!$transliterator instanceof \Transliterator) {
+                $transliterator = \Transliterator::create($transliterator);
+            }
+
+            if ($transliterator !== null) {
+                return $transliterator->transliterate($string);
+            }
         }
```

Comment:
this would mean to fall back to a method that may work completely different from the transliteration rule that is configured. We should probably throw an exception or at least warn about it. This is somewhat related to https://github.com/yiisoft/yii-core/issues/38#issuecomment-427301785 you basically want to disable intl if it does not work on your environment.